### PR TITLE
Merged two slider pages, merged slider Lvi and Fl

### PR DESCRIPTION
**Pull request by erenkan**

### CODE REVIEW

File: scripts/components/FlProductDetailSlider.ts [hunk 1]
```diff
@@ -1,37 +1,10 @@
-import System from '@smartface/native/device/system';
-import SwipeView from '@smartface/native/ui/swipeview';
 import FlProductDetailSliderDesign from 'generated/my-components/FlProductDetailSlider';
```
Comment (alimoncul) [inline scripts/components/FlProductDetailSlider.ts]: is this used

File: scripts/lib/listViewItemTypes.ts [hunk 2]
```diff
@@ -14,7 +14,8 @@ import LviHomeCategories from '../components/LviHomeCategories';
 
 import LviPdInfoSection from '../components/LviPdInfoSection';
 import LviPdTitleLikeSection from '../components/LviPdTitleLikeSection';
-import LviPdSlider from '../components/LviPdSlider';
+// import LviPdSlider from '../components/LviPdSlider';
```
Comment (alimoncul) [inline scripts/lib/listViewItemTypes.ts]: please remove comments

File: scripts/components/FlProductDetailSlider.ts (hunk 1 above)
Comment (erenkan) [reply scripts/components/FlProductDetailSlider.ts]: OK.

File: scripts/lib/listViewItemTypes.ts (hunk 2 above)
Comment (erenkan) [reply scripts/lib/listViewItemTypes.ts]: removed

**Pull request merged**